### ISSUE TITLE
isObject for json-unit-fluent

### ISSUE THREAD

**Issue by herau** [lukas-krecan/JsonUnit#15]
Hello

Could be cool to have a method to check that a node is a Json Object : 

```java
assertThatJson(json)
                .node("toto")
                .isObject()
```

what do you think ? 

I try to fork the repo to send you a pull request but when i fork your repo, the code is not update to date. (i haven't the method isArray for example).

**Comment by lukas-krecan** [comment]
It's strange that you do not see the newest code are you looking at https://github.com/lukas-krecan/JsonUnit/blob/master/json-unit-fluent/src/main/java/net/javacrumbs/jsonunit/fluent/JsonFluentAssert.java? I will add the isObject() method.

**Comment by herau** [comment]
I used The fork button on the project. Nevermind if you will implement The isObject method :-)

**Comment by lukas-krecan** [comment]
Released as 1.5.4, may take a while before it gets propagated to Maven repositories
